### ISSUE TITLE
LDAP Resolver + ldap3 2.4.1: Cannot find non-ASCII usernames

### ISSUE THREAD

**Issue by fredreichbier** [privacyidea/privacyidea#980]
Steps to reproduce:
* Current privacyIDEA master (c5fad91b4cc73245fdd4d46b13d3f968fda3d308), install ``ldap3==2.4.1``
* Add LDAP Resolver which contains user ``kölbel``
* Search for ``kö`` in user view

**Expectation**: We should find user ``kölbel``
**Current Behavior**: We find nothing.

We can also reproduce that in the REST API:

```
$ http GET http://localhost:5000/user/ PI-Authorization:$TOKEN realm=dc 'user=kölbel' | jq .result 
{
  "status": true,
  "value": [
    {
      "username": "kölbel",
      "surname": "Kölbel",
      ...
    }
  ]
}
$ http GET http://localhost:5000/user/ PI-Authorization:$TOKEN realm=dc 'user=kö*' | jq .result   
{
  "status": true,
  "value": []
}
```

**Comment by fredreichbier** [comment]
This will be fixed in the next release of ldap3, see cannatag/ldap3#509.

**Comment by fredreichbier** [comment]
Fixed in master by #1021 :tada:
